packages/discussions/graphql/resolvers/Discussion/comments.js

```diff
@@ -245,6 +244,29 @@ module.exports = async (discussion, args, context, info) => {
     }
   }
 
+  /* 
+    AUTO = comments are sorted 
+    - by date if the first comment was created in the last 24 hours 
+    - or by votes otherwise 
+    the property resolvedOrderBy is just needed when DiscussionOrder === AUTO
+  */
+  let resolvedOrderBy
+  if (orderBy === 'AUTO') {
+    const sortedComments = [...comments].sort(
+      (a, b) => Date.parse(a.createdAt) - Date.parse(b.createdAt),
+    )
+    if (sortedComments.length > 0) {
+      const firstCommentCreatedAt = Date.parse(sortedComments[0].createdAt)
+      const twentyFourHoursAgo = new Date(
+        new Date().getTime() - 24 * 60 * 60 * 1000,
```

Comment:
```suggestion
        new Date().getTime() - 1000 * 60 * 60 * 24, // 24 hours
```

Would flip multiplication (miliseconds to seconds to minutes to hours to a day) and append a comment for faster mental parsing.